postgres-appliance/configure_spilo.py

```diff
@@ -140,6 +141,30 @@ def deep_update(a, b):
         autovacuum_max_workers: 5
         autovacuum_vacuum_scale_factor: 0.05
         autovacuum_analyze_scale_factor: 0.02
+  {{#CLONE_WITH_WALE}}
+  method: clone_with_wale
+  clone_with_wale:
+    command: python3 /clone_with_s3.py --envdir "{{CLONE_WALE_ENV_DIR}}" --recovery-target-time="{{CLONE_TARGET_TIME}}"
+    recovery_conf:
+        restore_command: envdir "{{CLONE_WALE_ENV_DIR}}" /wale_restore_command.sh "%f" "%p"
+        recovery_target_timeline: latest
+        {{#USE_PAUSE_AT_RECOVERY_TARGET}}
+        pause_at_recovery_target: false
+        {{/USE_PAUSE_AT_RECOVERY_TARGET}}
+        {{^USE_PAUSE_AT_RECOVERY_TARGET}}
+        recovery_target_action: promote
+        {{/USE_PAUSE_AT_RECOVERY_TARGET}}
+        {{#CLONE_TARGET_TIME}}
+        recovery_target_time: "{{CLONE_TARGET_TIME}}"
+        {{/CLONE_TARGET_TIME}}
+  {{/CLONE_WITH_WALE}}
+  {{^CLONE_WITH_WALE}}
```

Comment:
`CLONE_WITH_BASEBACKUP` could only bet set if `CLONE_WITH_WALE` is not set.

Reply:
Unfortunately I couldn't find any way to express this in the template, and as the logic that guarantees that is in the Python code (that could be changed independently of the template) I thought of adding another check in the template itself. I see now it makes the template unreadable, so I'll get rid of it.